src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -8,16 +8,31 @@
   private User user;
   private List<Listing> businesses;
   
-  /** Create a new Small City Service instance
-  * @param mapLocation geolocation of user
-  * @return List of small local businesses
+  /** Create a new Small City Service instance from with no information **/
+  public SmallCityService() {
+
+  }
+  
+  /** 
+  * Create User instance from zipCode and get businesses list
+  * @param zipCode inputted zipcode of user
+  **/
+  public void createWithZip(String zipCode) {
```

Comment:
This is definitely a nit, and only if you want - you might change createWithZip() and createWithGeolocaton() to createUserWith... . Might make it a bit more clear for anyone looking at this for the first time

Reply:
Ah yes that's a lot clearer. Just changed.